Microsoft.Toolkit.Uwp.UI/Extensions/TitleBarExtensions.cs

```diff
@@ -0,0 +1,419 @@
+        /// </summary>
+        /// <param name="obj">The <see cref="DependencyObject"/> typically <see cref="Page"/></param>
+        /// <param name="value"><see cref="Color"/></param>
+        public static void SetForegroundColor(DependencyObject obj, Color value)
+        {
+            var titleBar = GetTitleBar();
+            if (titleBar != null)
+            {
+                titleBar.ForegroundColor = value;
+            }
+        }
+
+        /// <summary>
+        /// Gets <see cref="Color"/> for <see cref="ApplicationViewTitleBar.InactiveBackgroundColor"/>
+        /// </summary>
+        /// <param name="obj">The <see cref="DependencyObject"/> typically <see cref="Page"/></param>
+        /// <returns><see cref="Color"/></returns>
+        public static Color GetInactiveBackgroundColor(DependencyObject obj)
+        {
+            Color color;
+
+            var titleBar = GetTitleBar();
+            if (titleBar != null)
+            {
+                color = titleBar.InactiveBackgroundColor.GetValueOrDefault();
+            }
+
+            return color;
+        }
+
+        /// <summary>
+        /// Sets <see cref="Color"/> to <see cref="ApplicationViewTitleBar.InactiveBackgroundColor"/>
+        /// </summary>
+        /// <param name="obj">The <see cref="DependencyObject"/> typically <see cref="Page"/></param>
+        /// <param name="value"><see cref="Color"/></param>
+        public static void SetInactiveBackgroundColor(DependencyObject obj, Color value)
+        {
+            var titleBar = GetTitleBar();
+            if (titleBar != null)
+            {
+                titleBar.InactiveBackgroundColor = value;
+            }
+        }
+
+        /// <summary>
+        /// Gets <see cref="Color"/> for <see cref="ApplicationViewTitleBar.InactiveForegroundColor"/>
+        /// </summary>
+        /// <param name="obj">The <see cref="DependencyObject"/> typically <see cref="Page"/></param>
+        /// <returns><see cref="Color"/></returns>
+        public static Color GetInactiveForegroundColor(DependencyObject obj)
+        {
+            Color color;
+
+            var titleBar = GetTitleBar();
+            if (titleBar != null)
+            {
+                color = titleBar.InactiveForegroundColor.GetValueOrDefault();
+            }
+
+            return color;
+        }
+
+        /// <summary>
+        /// Sets <see cref="Color"/> to <see cref="ApplicationViewTitleBar.InactiveForegroundColor"/>
+        /// </summary>
+        /// <param name="obj">The <see cref="DependencyObject"/> typically <see cref="Page"/></param>
+        /// <param name="value"><see cref="Color"/></param>
+        public static void SetInactiveForegroundColor(DependencyObject obj, Color value)
+        {
+            var titleBar = GetTitleBar();
+            if (titleBar != null)
+            {
+                titleBar.InactiveForegroundColor = value;
+            }
+        }
+
```

Comment:
Ok. I am sorry. I checked again and it is the same class name `TitleBar` but it is another namespace. You can find use it in using `CoreApplication.GetCurrentView().TitleBar` and the class name is `CoreApplicationViewTitleBar`. It has 5 properties but I generally only use the `ExtendViewIntoTitleBar`.